src/swift/Wrapper.swift

```diff
@@ -188,21 +188,20 @@ extension DispatchSource : DispatchSourceProcess,
 
 internal class __DispatchData : DispatchObject {
 	internal let __wrapped:dispatch_data_t
-	internal let __owned:Bool
 
 	final internal override func wrapped() -> dispatch_object_t {
 		return unsafeBitCast(__wrapped, to: dispatch_object_t.self)
 	}
 
 	internal init(data:dispatch_data_t, owned:Bool) {
```

Comment:
that `owned` tag should really be `transferOwnership: Bool` but the patch LGTM and I should have realized when we reviewed the earlier patch :/